### PR TITLE
kbuild.jinja2: Add dtbs_check parameter

### PR DESCRIPTION
**Pull request by nuclearcat**
Add dtbs_check as parameter for kbuild

### CODE REVIEW

File: config/pipeline.yaml [hunk 1]
```diff
@@ -179,6 +179,15 @@ jobs:
   kbuild-gcc-10-arm64:
     <<: *kbuild-gcc-10-arm64-job
 
+  kbuild-gcc-10-arm64-dtbscheck:
+    <<: *kbuild-gcc-10-arm64-job
+    params:
+      arch: arm64
+      compiler: gcc-10
+      cross_compile: 'aarch64-linux-gnu-'
+      defconfig: defconfig
```
Comment (a-wai) [inline config/pipeline.yaml:188]:
```suggestion
      <<: *kbuild-gcc-10-arm64-params
```

File: config/pipeline.yaml [hunk 2]
```diff
@@ -384,6 +393,12 @@ scheduler:
   - job: kbuild-gcc-10-arm64
     <<: *build-k8s-all
 
+  - job: kbuild-gcc-10-arm64-dtbscheck
+    <<: *build-k8s-all
+    rules:
+      tree: next
+      branch: 'master'
```
Comment (a-wai) [inline config/pipeline.yaml:400]: Those should go into the job definition, not the scheduler entry (this could be a future TODO though).

**Review by a-wai — CHANGES_REQUESTED** [review]

File: config/pipeline.yaml [hunk 3]
```diff
@@ -179,6 +179,15 @@ jobs:
   kbuild-gcc-10-arm64:
     <<: *kbuild-gcc-10-arm64-job
 
+  kbuild-gcc-10-arm64-dtbscheck:
+    <<: *kbuild-gcc-10-arm64-job
+    params:
```
Comment (a-wai) [inline config/pipeline.yaml:184]:
```suggestion
    kind: test
    params:
```
`kbuild-gcc-10-arm64-job` has `kind: kbuild`, I think this one should be different

Comment (a-wai) [reply config/pipeline.yaml:185]: yes, that's what I meant, and also why we keep using the kbuild template; but the node/results should clearly be identified as `test` for easier understanding.

**Comment by nuclearcat** [comment]
Thanks all, merging

**Pull request merged**